web/blog/2024-09-17-from-idea-to-20k-in-days-how-wasp-accelerated-nuloapps-launch.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "From Idea to $20K in Days: How Wasp Accelerated NuloApp's Launch"
+authors: [milica]
+image: /img/nuloapp/wasp-friends.webp
```

Comment:
How important do you think the banner image is? I assumed for Dev.to and the like that a catchy banner makes a big difference. I always make them with some meme as a reference image, because there's so much context within the meme image that people understand right away.

Reply:
It is a core thing for SEO. Original content with original images gets x times more push than posts without one. These are also very important for being featured in those "suggested" posts on Google for mobile.